extensions/invite-notify-3.2.md

```diff
@@ -34,7 +34,7 @@ being invited.
 
 ## Example
 
-    :ChanServ!ChanServ@example.com INVITE Attila #channel
+    :ChanServ!ChanServ@example.com INVITE Attila +#channel
```

Comment:
I personally think it should also have an example of without prefix, otherwise the text beneath reads oddly I feel.